megatron/core/optimizer/qk_clip.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2024, NVIDIA CORPORATION. All rights reserved.
+
+import torch
+
+
+def clip_qk(model) -> float:
+    """
+    Clip the QK attention logits to the threshold, recommended for Muon optimizer.
+
+    Args:
+        model: The model to clip the QK attention logits, a list of model chunks.
+
+    Returns:
+        The maximum attention logit, a float.
+    """
+
+    log_max_attention_logit = 0
+    for model_chunk in model:
```

Comment:
i dont remember, is model always a list?

Reply:
According to https://github.com/NVIDIA/Megatron-LM/pull/1929/files#diff-7fc2a91343e49be3c1705b8237bbe437029132d8826e5c963e1b0b84d457bc38L1250. I think it is true for MLM, is it true for MBridge as well?